src/reducer/database/osm/models.ts

```diff
@@ -1,4 +1,10 @@
 //#region > Imports
+//> Moment
```

Comment:
Is there a need for a headline and a //#INSTALL?

Reply:
Yea, anchors are defined in https://github.com/snek-at/tonic, but this anchor right there does not fit the standard. I will adjust this. One sec.

Reply:
BTW. All other anchors will be set in the upcoming documentation update.